### PR TITLE
Integration

### PR DESCRIPTION
**Pull request by SyrianSpock**
This PR integrates the computer vision setup for the contest:
- adds try/catch and checks to avoid crashing when things go wrong
- stream results over UAR
- document some of the setup on the RPi Zero
- filter the color sequence to correct for detection errors
- defines a service to run the color detection directly after boot

### CODE REVIEW

**Comment by antoinealb** [comment]
I **strongly* suggest running the code as a systemd service to have it restart automatically in case of a crash or something else. I have installed one in the pi used for the belgium cup. I can't remember the exact name but a `find /etc -name camera.service` should allow you to find it. It is very simple, tweak it to your need.

It also makes it really easy to enable / disable the code by running `systemctl enable camera.service` to have it start on boot.

**Comment by SyrianSpock** [comment]
It works now, ready for review

File: detect.service [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[Unit]
```
Comment (antoinealb) [inline detect.service:1]: this is not the version on the pi, it lacks the wantedby line.

File: planReader/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+matplotlib
```
Comment (antoinealb) [inline planReader/requirements.txt:1]: maybe pin the versions ? (matplotlib==something.foo.bar)

File: detect.service (hunk 1 above)
Comment (SyrianSpock) [reply detect.service:1]: fuuu :sob:

File: planReader/requirements.txt (hunk 2 above)
Comment (SyrianSpock) [reply planReader/requirements.txt:1]: I'll let @doikovski select the version needed

Comment (antoinealb) [reply planReader/requirements.txt:1]: pip freeeze gives you the version info.

Comment (SyrianSpock) [reply planReader/requirements.txt:1]: true, but since we're using conda, I just don't want to mess with the setup

Comment (antoinealb) [reply planReader/requirements.txt:1]: conda 😭

**Comment by antoinealb** [comment]
LGTM

**Pull request merged**